packages/xo-web/src/xo-app/vm/tab-network.js

```diff
@@ -187,16 +198,23 @@ class VifAllowedIps extends BaseComponent {
           map(this._getIps(), (ip, ipIndex) => (
             <Row>
               <Col size={10}>
-                <XoSelect
-                  containerPredicate={this._getIsNetworkAllowed()}
-                  onChange={newIp => this._saveIp(ipIndex, newIp)}
-                  predicate={this._getIpPredicate()}
-                  resourceSetId={resourceSet}
-                  value={ip}
-                  xoType={resourceSet ? 'resourceSetIp' : 'ip'}
-                >
-                  {ip}
-                </XoSelect>
+                {resourceSet !== undefined && !isAdmin ? (
+                  <XoSelect
+                    containerPredicate={this._getIsNetworkAllowed()}
+                    onChange={newIp => this._saveIp(ipIndex, newIp)}
+                    predicate={this._getIpPredicate()}
+                    resourceSetId={resourceSet}
+                    value={ip}
+                    xoType='resourceSetIp'
+                  >
+                    {ip}
+                  </XoSelect>
+                ) : (
+                  <Text
+                    onChange={newIp => this._saveIp(ipIndex, newIp)}
+                    value={ip}
+                  />
+                )}
```

Comment:
I think IP pools currently can be used even without a resource set.

This PR is breaking this workflow.